aibolit/patterns/if_return_if_detection/if_detection.py

```diff
@@ -24,9 +11,15 @@ class CountIfReturn:
     def __init__(self):
         pass
 
-    def value(self, filename: str):
-        ''''''
-        tree = newJavalangImproved(filename)
-        if_decls = tree.filter([javalang.tree.IfStatement])
-
-        return if_decls
+    def value(self, filename: str) -> List[int]:
+        detected_lines = []
+        p = JavaPackage(filename)
+        for class_name in p.java_classes:
+            tree = p.java_classes[class_name]
```

Comment:
We better call it as is - `java_class`.

Reply:
Why do you think that it is better? To use java_class we need to initialize ast with parameters, such as tree, root, java_package. Hence, the work is only growth.